.travis.yml

```diff
@@ -114,6 +114,8 @@ script:
     else
       make -j$(nproc) check
     fi
+# Check for whitespace errors
+  - git diff --check $(git hash-object -t tree /dev/null)
```

Comment:
What do you think about moving this check all the way up to the top of the `script` section? Our ci loop is getting pretty long and failing early is good in these situations.

Reply:
Done